Library/Homebrew/env_config.rb

```diff
@@ -303,6 +303,10 @@ module EnvConfig
                       "or `$HOME/.homebrew/livecheck_watchlist.txt` otherwise.",
         default:      "#{ENV.fetch("HOMEBREW_USER_CONFIG_HOME")}/livecheck_watchlist.txt",
       },
+      HOMEBREW_LIVECHECK_AUTOBUMP:               {
```

Comment:
These need to be in alphabetical order to pass the "Homebrew::EnvConfig ENVS sorts alphabetically" test, so `HOMEBREW_LIVECHECK_AUTOBUMP` will need to come before `HOMEBREW_LIVECHECK_WATCHLIST`.

Reply:
Ah yes. I wrote that test. 😳 Proof that we can’t remember everything all the time!